### DISCUSSION TITLE
Remove Reviewers

### DISCUSSION THREAD

**Discussion by aaronflorey** [Mergifyio/mergify#3110] [Q&A]
I've got my workflow to request reviews from other developers once tests pass etc, but I can't find a way to remove those reviewers if something changes?

Should I make a feature request for this

**Comment by sileht** [comment]
Hi @aaronflorey I think you're are looking at: https://docs.mergify.io/actions/dismiss_reviews/, this will remove any reviews each time commits are updated or added.

**Reply by aaronflorey** [reply]
hey @sileht, Thanks for the reply.

Sorry if I was unclear, I'm wanting to remove the assigned reviewers so that it doesn't show up in their list until the PR is fixed

Thanks

**Comment by sileht** [comment]
Oh I see, we don't have such feature yet. Feel free to create an issue.
